event_monitor/src/lib.rs

```diff
@@ -3,59 +3,102 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// re-export flume
+pub use flume;
```

Comment:
Why do you need to re-export flume?

Reply:
The idea was that VMM would be able to use `flume` directly from the `event_monitor` crate when I get around to implementing #5560. But on second thought, I think this might not make sense since CH uses a single `Cargo.lock` file. I'll revert it, thanks for pointing it out.